src/Symfony/Component/HttpKernel/Exception/HttpException.php

```diff
@@ -21,6 +21,15 @@ class HttpException extends \RuntimeException implements HttpExceptionInterface
     private $statusCode;
     private $headers;
 
+    /**
+     * HttpException constructor.
+     *
+     * @param int             $statusCode An HTTP response status code
+     * @param string|null     $message    The internal exception message
+     * @param \Exception|null $previous   The previous exception
+     * @param array           $headers
```

Comment:
We could add something here to explain if these are the HTTP headers sent ... or the HTTP headers added to the default ones, etc.